lib/presentation/app_bar_login_register/widget/app_bar_log_reg.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:book/styles/app_colors.dart';
```

Comment:
package name is not good, you can create common and inside you can create book_app_bar widget